security/experimental_authenticators.rst

```diff
@@ -0,0 +1,432 @@
+Using the new Authenticator-based System
+========================================
+
+.. versionadded:: 5.1
+
+    The authenticator-based system was introduced as
+    :doc:`experimental feature </contributing/code/experimental>` in
+    Symfony 5.1.
+
+In Symfony 5.1, a new authentication system was introduced. This system
+changes the internals of the Security system, to make it more extensible
+and easier to understand.
+
+Enabling the System
+-------------------
+
+The authenticator-based system can be enabled using the
+``enable_authenticator_manager`` setting:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/security.yaml
+        security:
+            enable_authenticator_manager: true
+            # ...
+
+    .. code-block:: xml
+
+        <!-- config/packages/security.xml -->
+        <?xml version="1.0" encoding="UTF-8"?>
+        <srv:container xmlns="http://symfony.com/schema/dic/security"
+            xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+            xmlns:srv="http://symfony.com/schema/dic/services"
+            xsi:schemaLocation="http://symfony.com/schema/dic/services
+                https://symfony.com/schema/dic/services/services-1.0.xsd
+                http://symfony.com/schema/dic/security
+                https://symfony.com/schema/dic/security/security-1.0.xsd">
+
+            <config enable-authenticator-manager="true">
+                <!-- ... -->
+            </config>
+        </srv:container>
+
+    .. code-block:: php
+
+        // config/packages/security.php
+        $container->loadFromExtension('security', [
+            'enable_authenticator_manager' => true,
+            // ...
+        ]);
+
+The new system is backwards compatible with the current authentication
```

Comment:
We should maybe come up with a consistent name for the new system. "Guards" were very nice to use in a sentence. "Authenticator-based system" is both too technical and too long imho. "New system" on the other hand is too vague.

Reply:
If you use the word « new » maybe use the word « old » instead of « current » ?
Otherwise, what about « next » instead of new?
As i understood, it will be the new when this will be the only way, isnt it?

Reply:
It's the new system, but it's experimental so it's not really the "current" system. Decided to keep using "new" or "authenticator-based system" - at least for now.